start-sylius.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Start MySQL
+echo "Starting MySQL..."
+sudo service mysql start
+
+# Wait for MySQL to be ready
+sleep 2
+
+# Start PHP development server
+echo "Starting PHP development server on port 8000..."
+cd /workspaces/Sylius
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)